calblueprint/unloop
# Highlight required fields when creation fails & Pundit fix

Highlights required fields when creation fails. field are highlighted ONLY after user attempts to submit/create, and will remain highlighted as long as required field is empty. This PR also handles the Pundit error that occurred if a request is sent with no ActionItems or Participants.
 
## To Test:
Pull, run, and attempt to create Action Items without filling in required fields. Check for unusual behavior.

### Screenshots
Screenshot taken after hitting submit for a new particular action item
![image](https://user-images.githubusercontent.com/35501399/80066701-4a7e5700-84f1-11ea-8f62-c1627d98d553.png)

The left text would also become red if there were no participants selected and "ASSIGN" was pressed.
![image](https://user-images.githubusercontent.com/35501399/80066736-62ee7180-84f1-11ea-9f09-6d8693b28c4e.png)


CC: @didvi
